server/etcdserver/cluster_util.go

```diff
@@ -256,7 +256,14 @@ func isCompatibleWithVers(lg *zap.Logger, vers map[string]*version.Versions, loc
 			)
 			return false
 		}
+
+		// if downgrade is enabled, and one minor version down
+		// safe to not fail (e.g., local version 3.4, cluster version 3.5)
 		if maxV.LessThan(*clusterv) {
+			if allowDowngrade && membership.IsValidDowngrade(clusterv, maxV) {
```

Comment:
Let's be as verbose as possible, since it's critical code path.

```go
lg.Warn("marking the cluster version of remote member compatible to allow downgrade, can be unsafe",
  zap.String("remote-member-id", id),
  zap.String("remote-member-cluster-version", clusterv.String()),
  zap.String("minimum-cluster-version-supported", minV.String()),
)
```